src/main/webapp/js/community-page-loader.js

```diff
@@ -1,40 +1,52 @@
       /** Fetches users and adds them to the page. */
-      function fetchUserList() {
+      function fetchUserList() 
+      {
         const url = '/user-list';
         fetch(url).then((response) => {
           return response.json();
         }).then((users) => {
           const list = document.getElementById('list');
           list.innerHTML = '';
-
+		  
+          /*separating user link and about me section by " " */
           users.forEach((user) => {
-            let EndOfUsernameindex = user.indexOf(".com")+4;
-            const userListItem = buildUserListItem(user.substring(0, EndOfUsernameindex), user.substring(EndOfUsernameindex));
+            let endOfUsernameindex = user.indexOf(" ");
+            const userListItem = buildUserListItem(user.substring(0, endOfUsernameindex), user.substring(endOfUsernameindex+1));
             list.appendChild(userListItem);
           });
         });
       }
 
       /**
-       * Builds a list element that contains a link to a user page, e.g.
-       * <li><a href="/user-page.html?user=test@example.com">test@example.com</a></li>
+       * Builds a list element that contains:
+       * a link to a user page, e.g. and 
+       * displays user "about me" text if posted by user
+       * example:
+       * <li><a href="/user-page.html?user=test@example.com">test@example.com</a>
+       * <textarea readonly = 'readonly'> about me <textarea></li>
```

Comment:
Thanks for adding these! Could you also add what the params user and about me are expected to look like?